pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -40,12 +40,152 @@
         <Exclude>Compile</Exclude>
       </Dependency>
 
+      <!-- Only need the placeholder in the netcoreapp1.0 folder -->
       <File Include="$(PlaceholderFile)" >
         <SkipPackageFileCheck>true</SkipPackageFileCheck>
         <TargetPath>lib/netcoreapp1.0</TargetPath>
       </File>
     </ItemGroup>
   </Target>
 
+  <!--
+    Manually include the netcoreapp1.0 dependencies.
+    If any versions change in the release/1.0.0 branch they should
```

Comment:
Can we move all of the dependencies into the pkgproj like this, instead of doing the crazy regex-matching stuff above? If we did that, we wouldn't need to be translating from one to the other in a new version, merely adding new stuff.